libimage/runtime.go

```diff
@@ -23,9 +24,18 @@ var json = jsoniter.ConfigCompatibleWithStandardLibrary
 
 // tmpdir returns a path to a temporary directory.
 func tmpdir() string {
-	tmpdir := os.Getenv("TMPDIR")
+	var tmpdir string
+	defaultContainerConfig, err := config.Default()
+	if err == nil {
+		tmpdir, err := defaultContainerConfig.ImageCopyTmpDir()
```

Comment:
```suggestion
		tmpdir, err = defaultContainerConfig.ImageCopyTmpDir()
```

Otherwise the error is not in the same scope for logrus